ide-support/revsaveasstandalone.livecodescript

```diff
@@ -627,7 +627,7 @@ command revSaveAsMacStandalone pStack, pRestoreFileName, pFolder, @xStandalonePa
    
    -- OK-2008-03-24 : Bug 6171. Restructured a little to ensure that the icon can be either relative or absolute.
    -- If the icon cannot be found at the exact location, assume its relative to the stack and put the stack path before it.
-   if there is no file sStandaloneSettingsA["OSX,iconFile"] and sStandaloneSettingsA["OSX,iconFile"] is not empty then
+   if not (sStandaloneSettingsA["OSX,iconFile"] begins with "/") and sStandaloneSettingsA["OSX,iconFile"] is not empty then
```

Comment:
@livecodepanos this will be incorrect when building for mac on windows. I'm not sure if we have an `isRelativePath` function we can use from anywhere but the following should resolve the bug:

```
if sStandaloneSettingsA["OSX,iconFile"] is not empty then
     if there is no file sStandaloneSettingsA["OSX,iconFile"] or \
               (the folder is tStackPath and \
               there is a file (the folder & "/" & sStandaloneSettingsA["OSX,iconFile"])) then
          put tStackPath & "/" before sStandaloneSettingsA["OSX,iconFile"]
     end if
end if
```

Reply:
Good spot @montegoulding, thanks :)